apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -294,6 +329,7 @@ const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfo} = {
     },
   },
   ['math_modulo']: {
+    text: 'Modulo',
```

Comment:
Nit: “remainder of” better matches the text shown on the block.

Reply:
Updated!